### ISSUE TITLE
[ALWAYS OPEN] Add more explanations

### ISSUE THREAD

**Issue by swapagarwal** [swapagarwal/awesome-eli5#9]

**Comment by Kamilahsantos** [comment]
hello @swapagarwal I would like to add the explanation of what is front end and what is backend.

**Comment by will-commited** [comment]
Hi I would like to add what is front end and back end as the previous user did not put in the definitions. 
Front end: Is what the user sees, the design of a website and some languages.
Back end: Things the user cannot see, how the site works, updates and changes.
